### ISSUE TITLE
Matrix fields and Eager Loading

### ISSUE THREAD

**Issue by terryupton** [putyourlightson/craft-elements-panel#3]
Hey @bencroker - should this plugin pick up potential eager-loading opportunities within Matrix fields?
As I am not seeing this listed at all within the plugin, when I am using matrix fields with assets.

**Comment by bencroker** [comment]
Yeah it probably should. Can you give me an example of some template code that should be picked up that is not?

**Comment by terryupton** [comment]
This is a simple example, with an assets field in a matrix block.
I have a more complicated one in another project with all kinds of additional elements, like entries, categories, assets and images. So I can dig that out on Monday and share it.

But this example has an asset field called documents and with the following code nothing is picked up.
```
    {% for matrixBlock in entry.homework.all() %}
      <h2>{{ matrixBlock.heading }}</h2>
      <div>
        {% include '_includes/_partials/text-block.twig' with { content: matrixBlock.description } %}
      </div>
      
      {% if matrixBlock.documents.count %}
        <h3>Documents</h3>
        <ul class="last-child">
          {% for file in matrixBlock.documents.all %}
            <li class="mb-6">
              <a href="{{ file.url() }}"
                 target="_blank" rel="noopener noreferrer"
                 class="inline-flex flex-wrap // items-starts no-underline"
              >
                {{ svg('@icons/file-' ~ file.extension ~ '.svg', class="w-8 // h-8 mb-1 // fill-current // mr-2 // flex-shrink-0") }}
                <div class="flex-1 text-sm">
                  {{ file.title }}
                </div>
              </a>
            </li>
          {% endfor %}
        </ul>
      {% endif %}
    {% endfor %}
 ```

**Comment by bencroker** [comment]
Thanks fixed in 1.1.0.
